src/main/java/org/zaproxy/zap/extension/jwt/attacks/SignatureAttack.java

```diff
@@ -143,6 +143,34 @@ private boolean executePubliclyWellKnownHMacSecretAttack() {
         return false;
     }
 
+    /**
+     * Mis-matching the token signature and token data, to verify if the JWT implementation verifies the signature properly.
+     * A malicious user can exploit this vulnerability by supplying an arbitrary claim in the JWT payload to obtain 
+     * new privileges or impersonate other users
+     */
+    private boolean executeIncorrectSignatureAttack() {
+        JWTHolder cloneJWTHolder = new JWTHolder(this.serverSideAttack.getJwtHolder());
+        JSONObject payloadJSONObject = new JSONObject(cloneJWTHolder.getPayload());
+        payloadJSONObject.put("username", "admin");
```

Comment:
Please add username and admin as string constants.

Reply:
Updated @preetkaran20